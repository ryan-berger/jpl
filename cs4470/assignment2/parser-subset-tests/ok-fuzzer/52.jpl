read image "L" to u1V.
let V = 6999180732.1
write image 81368 to ""
print ""
print "]zV"
read image "t" to p3I_2M_6a_6U.7v.
write image 1 to ""
print ":;Q+"
time time time print ""
print ""
show S2i_()
time assert g, ""
return 57392
time print "9^^VR7"
show R8m.4g.6x_3M_()
show M4O_
