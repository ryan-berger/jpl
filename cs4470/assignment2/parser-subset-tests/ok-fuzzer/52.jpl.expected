(ReadImageCmd "L" (VarArgument u1V.))
(StmtCmd (LetStmt (ArgLValue (VarArgument V)) (FloatExpr 6999180732)))
(WriteImageCmd (IntExpr 81368) "")
(PrintCmd "")
(PrintCmd "]zV")
(ReadImageCmd "t" (VarArgument p3I_2M_6a_6U.7v.))
(WriteImageCmd (IntExpr 1) "")
(PrintCmd ":;Q+")
(TimeCmd (TimeCmd (TimeCmd (PrintCmd ""))))
(PrintCmd "")
(ShowCmd (CallExpr S2i_))
(TimeCmd (StmtCmd (AssertStmt (VarExpr g) "")))
(StmtCmd (ReturnStmt (IntExpr 57392)))
(TimeCmd (PrintCmd "9^^VR7"))
(ShowCmd (CallExpr R8m.4g.6x_3M_))
(ShowCmd (VarExpr M4O_))
